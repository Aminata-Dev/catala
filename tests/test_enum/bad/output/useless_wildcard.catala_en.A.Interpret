[WARNING] Unreachable match case, all constructors of the enumeration E are already specified

  --> tests/test_enum/bad/useless_wildcard.catala_en
   | 
17 |     -- anything : 31
   |        ^^^^^^^^^^^^^
   + Useless wildcard
[RESULT] Computation successful! Results:
[RESULT] x = Case1 ()
[RESULT] y = 42
