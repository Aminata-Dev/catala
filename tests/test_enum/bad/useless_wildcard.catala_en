## Useless wildcard

```catala
declaration enumeration E:
  -- Case1
  -- Case2

declaration scope A:
  context output x content E
  context output y content integer

scope A:
  definition x equals Case1
  definition y equals match x with pattern
    -- Case1 : 42
    -- Case2 : 42
    -- anything : 31
```

```catala-test {id="A.Interpret"}
catala Interpret -s A
```
