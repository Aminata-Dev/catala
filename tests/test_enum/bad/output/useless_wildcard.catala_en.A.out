[ERROR] Unreachable match case, all constructors of the enumeration E are already specified.
[ERROR] 
[ERROR]   --> test_enum/bad/useless_wildcard.catala_en
[ERROR]    | 
[ERROR] 17 |     -- anything : 31
[ERROR]    |        ^^^^^^^^^^^^^
[ERROR]    + Useless wildcard
